homeassistant/components/switch/__init__.py

```diff
@@ -105,6 +109,16 @@ def today_power_mw(self):
         return None
 
     @property
+    def standby_state(self):
+        """ Is the device on - or in standby. """
+        return None
+
+    @property
+    def sensor_state(self):
+        """ Is the sensor on or off. """
```

Comment:
The insight switch starts as a regular switch with 'Switch = On or Off'

It alo monitors power usage in the 'On' state against a defined threshold.

```
Switch /  Power > Threshold / Overall - Status
 On          True                    On
 On          False                   Standby
 Off         False                   Off
````

The WeMo app displays this a Switch which has states On and Off and a Status Lamp

```
Switch /  Power > Threshold / WeMo Switch / WeMo lamp
 On          True                  On        Green
 On          False                 On        Amber
 Off         False                 Off       Off
````

While you could display this as two booleans - I think this is confusing in the On/On state where 'Standby is False'.

```
Switch /  Power > Threshold / Switch bool / Standby bool
 On          True                  True        False
 On          False                 True        True
 Off         False                 False       False
````

Which is why I went for the text version where the three state are shown as text - and None if the idea isn't supported.

Happy to change if you have a preferred option.